contracts/scripts/migrations/001-deploy-project-registry.ts

```diff
@@ -5,15 +5,19 @@ import { LedgerSigner } from "@anders-t/ethers-ledger";
 async function main() {
   const network = await ethers.provider.getNetwork();
   const networkName = await hre.network.name;
-
-  // without hardware wallet
-  // const [account] = await ethers.getSigners();
-  // const accountAddress = account.address;
-
-  // with hardware wallet
-  const account = new LedgerSigner(ethers.provider);
-  const accountAddress = await account.getAddress();
-
+  let account;
+  let accountAddress;
+
+  if(process.env.USE_HARDWARE_WALLET==='true') {
```

Comment:
I would change single `'` to `"`, just a small change to keep it like the other strings